src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
So if I'm reading you correctly, your proposal is:
1. Remove the functions altogether
2. Make `_renderNow` public so that it can be flipped at leisure, and rename it to something more understandable.

If so, I think that might require either having the user be aware of the need to call `syncDataToFrames()`, or to flip the variable to true? Kind of why I went for the functions: cheaper than making the var be observable, and allowing to sneak some extra functionality in.

But fair enough, I kinda realized I dropped the ball with my approach, because the var's being flipped back on both after each multinode/multiedge update.